.github/ISSUE_TEMPLATE/technical.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Tech
+about: A task, enhancement idea, or bug report related to the technical aspects of the project.
+title: A concise description of the issue
```

Comment:
```suggestion
title: Title of the issue description
```

Reply:
@steveeJ Since you have asked for modifications, please don't forget to mark your review as "changes requested." This triggers automation steps.

Reply:
My comments were meant as suggestions and not as change requests.